src/disco/gui/fd_gui_printf.c

```diff
@@ -1002,8 +1002,15 @@ fd_gui_printf_slot( fd_gui_t * gui,
         else                                                 jsonp_ulong( gui, "failed_transactions", slot->failed_txn_cnt );
         if( FD_UNLIKELY( slot->compute_units==ULONG_MAX ) ) jsonp_null( gui, "compute_units" );
         else                                                jsonp_ulong( gui, "compute_units", slot->compute_units );
-        if( FD_UNLIKELY( slot->fees==ULONG_MAX ) ) jsonp_null( gui, "fees" );
-        else                                       jsonp_ulong( gui, "fees", slot->fees );
+        if( FD_UNLIKELY( slot->transaction_fee==ULONG_MAX || slot->priority_fee==ULONG_MAX ) ) {
+          jsonp_null( gui, "fees" );
+          jsonp_null( gui, "transaction_fees" );
+          jsonp_null( gui, "priority_fees" );
+        } else {
+          jsonp_ulong( gui, "fees", slot->transaction_fee+slot->priority_fee ); // TODO Include JITO, rent?
+          jsonp_ulong( gui, "transaction_fees", slot->transaction_fee );
```

Comment:
Can you add these to the `websocket.md` documentation?

Reply:
Oops.  Will do.